docs/reference-guides/core-blocks.md

```diff
@@ -51,7 +51,7 @@ Prompt visitors to take action with a group of button-style links. ([Source](htt
 -	**Name:** core/buttons
 -	**Category:** design
 -	**Supports:** align (full, wide), anchor, spacing (blockGap, margin)
--	**Attributes:** 
+-	**Attributes:**
```

Comment:
One of my linting tools must have removed these trailing spaces automatically. They shouldn't be there anyway, so I guess it's fine to leave these changes in this PR?